components/AlumniCarousel.jsx

```diff
@@ -1,27 +1,46 @@
-import Image from "next/image";
-import { ALUMNI } from "../data/alumni";
-import { SwiperSlide } from "swiper/react";
-import { faLinkedin } from "@fortawesome/free-brands-svg-icons";
-import Carousel from "./Carousel";
-import FontAwesomeLink from "./FontAwesomeLink";
-import "swiper/css";
-import "swiper/css/navigation";
+import Image from 'next/image';
+import { SwiperSlide } from 'swiper/react';
+import { faLinkedin } from '@fortawesome/free-brands-svg-icons';
+import Carousel from './Carousel';
+import FontAwesomeLink from './FontAwesomeLink';
+import 'swiper/css';
+import 'swiper/css/navigation';
 
-const CAROUSEL_CONTAINER = "text-white rounded-lg w-[20rem] h-[44rem] shadow-2xl md:w-4/5 swiper-button-white";
+const CAROUSEL_CONTAINER = 'text-white rounded-lg w-[20rem] h-[44rem] shadow-2xl md:w-4/5 swiper-button-white';
 const ALUMNI_TILE =
-  "bg-[#7055FD] w-[20rem] h-[70rem] md:h-[44rem] rounded-lg items-center p-14 lg:p-10 flex flex-col shadow-2xl md:w-full md:flex-row";
-const ALUMNI_PROFILE = "flex flex-col items-center mx-auto text-center md:p-10";
+  'bg-[#7055FD] w-[20rem] h-[70rem] md:h-[44rem] rounded-lg items-center p-14 lg:p-10 flex flex-col shadow-2xl md:w-full md:flex-row';
+const ALUMNI_PROFILE = 'flex flex-col items-center mx-auto text-center md:p-10';
 
-const AlumniTile = ({ name, file, position, prevRole, testimonial, linkedin }) => {
+const keyStr = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=';
```

Comment:
question: what's this for

Reply:
This is just some code I copied from the NextJS official examples: https://github.com/vercel/next.js/blob/canary/examples/image-component/app/color/page.tsx

Apparently, what it does is it generates a DataURL for a 1x1 pixel image as a placeholder while the image is fetched and loaded. The keyStr contains all the characters in a 64-bit encoding which is used to convert an RGB value to binary data.